src/utils/utils.ts

```diff
@@ -111,13 +111,36 @@ export const getAllScriptURLParameters = (
   return params
 }
 
-export const getTabsData = (productsData: Product[]): TabItemProps[] => {
-  const tabs = productsData.map(product => ({
-    id: product.id,
-    label: product.type,
-    contents: getProductContent(product),
-  }))
-  return tabs
+export const getProductData = (productsData: Product[]): ProductItemProps[] => {
+  const products = productsData.reduce((acc, productItem) => {
+    const indexOfProductType = acc.findIndex(
+      item => item.productType === productItem.type
+    )
+    if (indexOfProductType >= 0) {
+      acc[indexOfProductType].productItems.push({
+        id: productItem.id,
+        label: createTabLabel(productItem.termPeriod),
+        contents: getProductContent(productItem),
+      })
+    } else {
+      acc.push({
+        productType: productItem.type,
+        productItems: [
+          {
+            id: productItem.id,
+            label: createTabLabel(productItem.termPeriod),
+            contents: getProductContent(productItem),
+          },
+        ],
+      })
+    }
+    return acc
+  }, [] as ProductItemProps[])
+  return products
+}
```

Comment:
the most intense change in this PR ☝️ 

Try to convert `productsData` that looks like this 👇 

> [
        {
            "id": "122",
            "type": "Standard",
            "interestFreePeriod": 18,
            "termPeriod": 18,
            "termRate": 0,
            "deferredPeriod": 0,
            "establishmentFee": 50,
            "description": "18 MTH INT FREE",
            "interestFreeMonthlyRepayment": 180,
            "minTermMonthlyRepayment": 30,
            "remainderInterestPeriodMontlyRepayment": 145
        },
        {
            "id": "123",
            "type": "Standard",
            "interestFreePeriod": 12,
            "termPeriod": 12,
            "termRate": 0,
            "deferredPeriod": 0,
            "establishmentFee": 50,
            "description": "12 MTH INT FREE",
            "interestFreeMonthlyRepayment": 200,
            "minTermMonthlyRepayment": 35,
            "remainderInterestPeriodMontlyRepayment": 165
        },
        {
            "id": "124",
            "type": "Fixed",
            "interestFreePeriod": 12,
            "termPeriod": 12,
            "termRate": 0,
            "deferredPeriod": 0,
            "establishmentFee": 50,
            "description": "12 MTH FIX",
            "interestFreeMonthlyRepayment": 200,
            "minTermMonthlyRepayment": 100,
            "remainderInterestPeriodMontlyRepayment": 0
        },
        {
            "id": "125",
            "type": "Hybrid",
            "interestFreePeriod": 12,
            "termPeriod": 24,
            "termRate": 16.99,
            "deferredPeriod": 0,
            "establishmentFee": 50,
            "description": "12 MTH INT FREE, 12 MTH FIX",
            "interestFreeMonthlyRepayment": 100,
            "minTermMonthlyRepayment": 35,
            "remainderInterestPeriodMontlyRepayment": 165
        }
    ]

to something that looks like this 👇 
![image](https://user-images.githubusercontent.com/63985172/136126754-63f9d167-4fa7-48bf-a9a6-036efef9da8e.png)

Reply:
Nice, well done 👍